### PR TITLE
Implement Horizontal Slider for Related Products

### PR DESCRIPTION
**Pull request by cschucode**
Used jQuery library to implement horizontal slider. Chevrons disappear when sliding left or right is no longer available.

Also,
* Added styles to Card component
* Fixed padding on Star Rating component
* Added preview thumbnails when hovering over card
* Added event handler to switch card images with thumbnails

[Trello](https://trello.com/c/UOELdPg3/49-create-carousel-slider-for-related-products)

### CODE REVIEW

File: client/components/relatedProducts/Card.jsx [hunk 1]
```diff
@@ -5,30 +5,74 @@ import { AiOutlineStar } from 'react-icons/ai';
...
+  };
+
   return (
-    <div className="related-products__card" role="button" tabIndex={0} onKeyPress={() => { handleCardClick(relatedProduct.id); }} onClick={() => { handleCardClick(relatedProduct.id); }}>
-      <div className="related-products__card-image" style={imageUrl ? { backgroundImage: `url(${imageUrl})` } : {}}>
-        <span className="related-products__action-btn" role="button" tabIndex={0} onKeyPress={() => { handleActionBtnClick(relatedProduct); }} onClick={() => handleActionBtnClick(relatedProduct)}>
-          <AiOutlineStar size={STAR_SIZE} />
+    <div className="related-products__card" role="button" tabIndex={0} onKeyPress={() => { handleCardClick(relatedProduct.id); }} onClick={() => handleCardClick(relatedProduct.id)}>
+      <div className="related-products__card-image" style={mainImageUrl ? { backgroundImage: `url(${mainImageUrl})` } : {}}>
+        <span className="related-products__action-btn" role="button" tabIndex={0} onKeyPress={() => { handleActionBtnClick(relatedProduct); }} onClick={(e) => { e.stopPropagation(); handleActionBtnClick(relatedProduct); }}>
```
Comment (runhui2010) [inline client/components/relatedProducts/Card.jsx]: Put the attributes in a new line might be more readable, and something will break the max 100 characters a line rule.

File: client/css/relatedProducts/ComparisonModal.scss [hunk 2]
```diff
@@ -1,19 +1,19 @@
-$modal-width: 450px;
+$modal-width: 50%;
```
Comment (runhui2010) [inline client/css/relatedProducts/ComparisonModal.scss:1]: is this the way to create a CSS variable? does it require importing JQuery?

Comment (runhui2010) [reply client/css/relatedProducts/ComparisonModal.scss:1]: Yeah, I forget this file is SCSS.

**Review by runhui2010 — APPROVED** [review]
Clean codes, can't wait to see your image slider.

File: client/css/relatedProducts/ComparisonModal.scss (hunk 2 above)
Comment (cschucode) [reply client/css/relatedProducts/ComparisonModal.scss:1]: Yep. They can be useful. Kind of like self-documenting code. 
- https://sass-lang.com/documentation/variables

**Pull request merged**